Apps/WebClient/src/ClientApp/src/components/communication.vue

```diff
@@ -26,18 +26,14 @@ import { ICommunicationService } from "@/services/interfaces";
 
 @Component
 export default class CommunicationComponent extends Vue {
-  private communication?: Communication = null;
+  private communicationText: string = "";
```

Comment:
having the communication object sounds right, it can be undefined at the begining so

```
 private communication?: Communication;
private get text(): string {
    return this.communication?.text || "";
  }
...

Reply:
the communication needs an initial value of null otherwise the computed properties won't work properly, not sure if this is a bug with vue but i was struggling with it for a while.